### PR TITLE
Update OpenAPI

### PR DESCRIPTION
**Pull request by fjakobs**
This PR adds support for
- SQL execution API
- model serving API

### CODE REVIEW

File: packages/databricks-sdk-js/src/services/wsfs/WorkspaceFsDir.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export class WorkspaceFsDir extends WorkspaceFsEntity {
                 {
                     path: validPath,
                     overwrite,
-                    format: "AUTO",
```
Comment (kartikgupta-db) [inline packages/databricks-sdk-js/src/services/wsfs/WorkspaceFsDir.ts:98]: We need it to be auto to correctly import notebooks. I added it manually. If you are already updating it, maybe you can add the type to openapi as well?

File: packages/databricks-sdk-js/src/services/wsfs/WorkspaceFs.integ.ts [hunk 2]
```diff
@@ -94,7 +94,7 @@ describe(__filename, function () {
 
         const content = await integSetup.client.workspace.export({
             path: file.details.path,
-            format: "AUTO",
+            format: "SOURCE",
```
Comment (kartikgupta-db) [inline packages/databricks-sdk-js/src/services/wsfs/WorkspaceFs.integ.ts:97]: same

File: packages/databricks-sdk-js/src/apis/workspace/model.ts [hunk 3]
```diff
@@ -46,13 +46,7 @@ export interface Export {
...
  * value is case sensitive.
  */
-export type ExportFormat =
-    | "AUTO"
-    | "DBC"
-    | "HTML"
-    | "JUPYTER"
-    | "R_MARKDOWN"
-    | "SOURCE";
+export type ExportFormat = "DBC" | "HTML" | "JUPYTER" | "R_MARKDOWN" | "SOURCE";
```
Comment (kartikgupta-db) [inline packages/databricks-sdk-js/src/apis/workspace/model.ts:49]: Just need to revert this for manual update for now.

Comment (fjakobs) [reply packages/databricks-sdk-js/src/apis/workspace/model.ts:49]: I was wondering about it already. Do you know how to get this change upstream?

Comment (fjakobs) [reply packages/databricks-sdk-js/src/apis/workspace/model.ts:49]: Done

File: packages/databricks-sdk-js/src/services/wsfs/WorkspaceFsDir.ts (hunk 1 above)
Comment (fjakobs) [reply packages/databricks-sdk-js/src/services/wsfs/WorkspaceFsDir.ts]: reverted that change

**Review by kartikgupta-db — APPROVED** [review]

**Pull request merged**